### DISCUSSION TITLE
ポエムについて積極的な対策を打って欲しい

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [increments/qiita-discussions#438] [要望・アイデア・不具合]
以下、ご検討願います。初めて投稿するため作法に沿っていない部分がもしあった場合ご指摘ください。

## 私の主張
* 昨今ではQiitaにポエムが氾濫しているように感じる
  * しかし、全体に占める`tag:ポエム`の件数は0.6%程度である[^2]。
* 特定の話題でGoogle検索などをかけるとQiitaのポエム記事がヒットすることがたまにあり、やるせない気持ちになる。
* ポエムの存在は「エンジニアにとって再利用性・汎用性の高い情報が集まる場をつくろう」という[コミュニティガイドライン][com]の目標と両立しない

[^2]: 2023年2月21日時点では836461件中4743件。

[com]: https://help.qiita.com/ja/articles/qiita-community-guideline

#### 再利用性や汎用性に欠ける例
下記観点に主軸を置いた記事は、全て[コミュニティガイドライン][com]の「エンジニアにとって再利用性・汎用性の高い情報が集まる場をつくろう」に違反していると考える。なお、下記では例をあげることがあるが、その著者を批判する意図はない。

* 著者が持つ主義
* 著者自身の主張
  * https://qiita.com/aki_senri/items/f785e4046261cf14a9c3
* 自分の経験のうち、エンジニアリング(典型的にはプログラミングだが、それに限定されるものではないと解釈される)に関連しないもの
* Qiitaに置けるかどうか怪しい記事 (cf. #231)
  * [tag:ポエム tag:入試問題](https://qiita.com/search?sort=&q=tag%3A%E3%83%9D%E3%82%A8%E3%83%A0+tag%3A%E5%85%A5%E8%A9%A6%E5%95%8F%E9%A1%8C)
  * [tag:ポエム tag:確定申告](https://qiita.com/search?q=tag%3A%E3%83%9D%E3%82%A8%E3%83%A0+tag%3A%E7%A2%BA%E5%AE%9A%E7%94%B3%E5%91%8A)
  * [tag:ポエム tag:読書感想文](https://qiita.com/search?q=tag%3A%E3%83%9D%E3%82%A8%E3%83%A0+tag%3A%E8%AA%AD%E6%9B%B8%E6%84%9F%E6%83%B3%E6%96%87)
  * …
* Qiitaの記事に対する反論
  * このカテゴリに属する記事の立ち位置がガイドライン上不明瞭という別の問題がある。ここでは違反しているものと考える。

#### 再利用性や汎用性を満たしている例
これらの記事は基本的にポエムタグを使用するべきではないと考える。

##### 例
記事：https://qiita.com/mayocorn/items/4edff486428240864808
要約：Qiita利用者がAtCoderにおいて青色レートに入るまでの道筋
なぜ再利用性や汎用性を満たしていると考えるか：記事冒頭で「ITに関してはSESで半年ほど働いた経験あり」としながら、レーティングの順位付における上位数%のランク[^1]に至るまで行った行為を記述しているため。

[^1]: 少し古いデータだが、2022年6月にAtCoderの全ユーザーを対象とした解析を行った[記事](https://gogocatslife.com/20220619atcoderhist)が存在する

#### 両方に当てはまる例
この場合、ポエムに該当する部分は「再利用性・汎用性の高い情報」へのノイズとなるため消去されるべきであると考える。

## 私の提案
* ポエムを投稿する際にはポエムタグをつけることを必須化する
  * idealではないが、タグミュートという既存の枠組みを使って回避することができると考える
* ポエムタグの全件精査を行う
* 必要に応じてコミュニティガイドラインの改訂を行う

※ポエムタグを廃止するという提案を出すことも考えましたが、その方法だとやはり止まらない利用者は止まらないため、棲み分けができるという実利を優先しました。

**Comment by tanaka-asami415** [comment]
@KisaragiEffective 
貴重なご意見をいただきありがとうございます。

仰っている内容は、理屈としても感情としても概ね理解できます。
運営としても、ガイドラインに沿っていない記事が多く存在していることや、それらがよく表示されていることは課題に感じています。

しかしながら、基準は人それぞれ異なるため、該当の記事が一様にエンジニアに役立たないと判断することは難しく[^1]
ご提案いただいた必須化を行ったとしても解決することは難しいと考えています。

そのため、運営側として一律で禁止することはせず、ユーザーさまの判断に委ねている状況です。

一律で禁止するのではなく、タグのミュートなどの方法で、受け取りたい情報をそれぞれの人が選ぶ世界にしたいと思っています。
将来的にはフィードを技術とそれ以外でわけることもあるかもしれません。
（申し訳ありませんが、現時点での確約はできません。）

今回いただいたご意見をすぐに解決することができず申し訳ありません。
運営としても改善したいと思っているのですが、今ある記事や使ってくださっている皆様にとって最大多数の選択が何かは迷っていることをご理解いただけますと幸いです🙇

[^1]:2021年4月にガイドラインを改定し、エンジニアにとって「役立つ」ような内容であれば、様々な内容の投稿をしていただけるようになりました。
↓詳細はこちらをご参照ください。
https://blog.qiita.com/notice-community-guideline-update/
